Packs/CheckPointHEC/Integrations/CheckPointHEC/test_data/checkpointhec-get_email_info.json

```diff
@@ -0,0 +1,113 @@
+{
+  "responseEnvelope": {
+    "requestId": "b58b1e41-1018-4062-9d1c-bcaeccbfcb93",
+    "responseCode": 200,
+    "responseText": "",
+    "additionalText": "",
+    "recordsNumber": 1,
+    "scrollId": ""
+  },
+  "responseData": [
+    {
+      "entityInfo": {
+        "entityId": "637d86da7bcf42375cb8431d266e3dc3",
+        "customerId": "fdolab",
+        "customerOem": "Avanan",
+        "saas": "office365_emails",
+        "saasEntityType": "office365_emails_email",
+        "entityCreated": "2022-08-15T21:24:27.745655Z",
+        "entityUpdated": "2022-08-15T21:24:36.979329",
+        "entityActionState": null
+      },
+      "entityPayload": {
+        "fromEmail": "fernandom@checkpoint.com",
```

Comment:
We don't recommend using real email addresses or URL's, so if you have any email or url that is real, please change it to be a dummy one.
Please check all the test_data and test files.